changelog/29131.txt

```diff
@@ -1,3 +1,4 @@
 ```release-note:change
```

Comment:
Should this be a bug as well?

Reply:
Good call, forgot that this came from the Actions Bot. Updated!